spec/system/placements/persona_user_visits_the_placements_start_page_spec.rb

```diff
@@ -0,0 +1,58 @@
+require "rails_helper"
+
+RSpec.describe "Persona user visits the placements start page", :persona_sign_in, service: :placements, type: :system do
+  include ActionView::Helpers::SanitizeHelper
+
+  scenario do
+    given_i_am_on_the_placements_service_start_page
+    then_i_see_the_start_page_detail
+    and_i_see_the_start_now_button
+
+    when_i_click_on("Start now")
+    then_i_can_see_the_persona_sign_in_button
+  end
+
+  private
+
+  def given_i_am_on_the_placements_service_start_page
+    # The user does not need to sign in, and we need to navigate somewhere to start the process
+    visit placements_root_path
+
+    expect(page).to have_title("Manage school placements - GOV.UK")
+    expect(page).to have_h1("Manage school placements")
+    expect(page).to have_element(:p, text: "Use this service if you are involved in arranging school placements for trainee teachers in England.", class: "govuk-body-m")
```

Comment:
I made this comment on jamies PR; might be worth having a have_govuk_body matcher with an optional variable for size (s,m,l). It comes up fairly frequently